src/python/grpcio/grpc/experimental/aio/_typing.py

```diff
@@ -13,17 +13,18 @@
 # limitations under the License.
 """Common types for gRPC Async API"""
 
-from typing import (Any, AnyStr, AsyncIterable, Callable, Iterable, Sequence,
-                    Tuple, TypeVar, Union)
+from typing import (Any, AsyncIterable, Callable, Iterable, Sequence, Tuple,
+                    TypeVar, Union)
 
 from grpc._cython.cygrpc import EOF
+from ._metadata import Metadata, MetadataKey, MetadataValue
 
 RequestType = TypeVar('RequestType')
 ResponseType = TypeVar('ResponseType')
 SerializingFunction = Callable[[Any], bytes]
 DeserializingFunction = Callable[[bytes], Any]
-MetadatumType = Tuple[str, AnyStr]
-MetadataType = Sequence[MetadatumType]
+MetadatumType = Tuple[MetadataKey, MetadataValue]
```

Comment:
`MetadataKey` is pointing to `Text` which is used for Python 2/3 compatibility. Shall we change it to `str` to be more accurate in Python 3?

Reply:
Done, now ``MetadataKey`` points to ``str``